03Lesson/01Project/index.html

```diff
@@ -1,9 +1,71 @@
 <!DOCTYPE html>
 <html>
-<head>
-  <title></title>
-  <link rel="stylesheet" href="css/style.css">
-</head>
-<body>
-</body>
+  <head>
+    <title>First Project Intel</title>
+    <link rel="stylesheet" href="css/style.css" type="text/css">
+    <meta charset="utf-8">
+  </head>
+  <body>
+    <main>
+      <div class="topbanner">
+        <h1>
+          Intel
+        </h1>
+        <a href="http://www.intel.com/content/www/us/en/processors/core/core-i7-processor.html">
+          i7 Processor
+        </a>
+        <a href="http://www.intel.com/content/www/us/en/solid-state-drives/solid-state-drives-ssd.html">
+          Solid-State Drive
+        </a>
+      </div>
+      <h3>
+        Intel&reg; Core i7&trade;
+      </h3>
+      <p>
+        The Sixth Generation Intel&reg; Core&trade; i7 processors deliver a new class of computing with a host of new features to power the next generation of desktops, laptops, and 2 in 1 PCs. Expect lighting speeds a peak performance through even the toughest of tasks and games.
+      </p>
+      <article class="thirdbanner">
+        <section class="txtleft">
+            <h2>
+              Ready. Wake. Create.
+            </h2>
+            <p>
+              From zero to power in a flash, the 6th Generation Intel&reg; Core&trade; processor wakes quicker than ever and lets you switch in an out of apps more seamlessly than ever.
+            </p>

```

Comment:
This text is overflowing out of its container and being covered up by laptop image - this is because in your css you have:
`p, div {
    width: 700px;
    font-size: 11px;
    color: black;
    font-family: sans-serif;
}`
Can you think of how to refactor this styling so the overflow issue is fixed?